.gitignore

```diff
@@ -2,3 +2,5 @@ venv/
 env/
 **__pycache__**
 **egg**
+.pytest_cache/
+.git/
```

Comment:
the .git really  shouldnt be needed...

Reply:
I think I added it to make sure the .git files where not transferred to psbuild when calling 'make sync' to rsync local files to remote server.